src/cmd/support_archive/resources_test.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const testOperatorNamespace = "dynatrace"
+const manifestExtension = ".yaml"
```

Comment:
could reuse the `const ManifestsFileExtension = ".yaml"` instead

Reply:
I knew that this will come up, my opinion here is NO. Why? changing the const `ManifestsFileExtension` unintentionally will lead to passing unit tests, even though if its not intented